Sources/docc/DocCDocumentation.docc/linking-to-symbols-and-other-content.md

```diff
@@ -0,0 +1,198 @@
+# Linking to Symbols and Other Content
+
+Facilitate navigation between pages using links.
+
+## Overview
+
+DocC supports the following link types to enable navigation between pages:
+
+| Type | Usage |
+| --- | --- |
+| Symbol | Links to a symbol's reference page in your documentation. |
+| Article | Links to an article or API collection in your documentation catalog. |
+| Tutorial | Links to a tutorial in your documentation catalog. |
+| Web | Links to an external URL. |
+
+### Navigate to a Symbol
+
+To add a link to a symbol, wrap the symbol's name in a set of double backticks 
+(\`\`).
+
+```markdown
+``SlothCreator``
+```
+
+For nested symbols, include the path to the symbol in the link.
+
+```markdown
+``SlothCreator/Sloth/eat(_:quantity:)``
+```
+
+DocC resolves symbol links relative to the context they appear in. For example, 
+a symbol link that appears inline in the `Sloth` class, and targets a 
+symbol in that class, can omit the `SlothCreator/Sloth/` portion of the symbol 
+path.
+
+In some cases, a symbol's path isn't unique, such as with overloaded methods in 
+Swift. For example, consider the `Sloth` structure, which has multiple 
+`update(_:)` methods.
+
+```swift
+/// Updates the sloth's power.
+///
+/// - Parameter power: The sloth's new power.
+mutating public func update(_ power: Power) {
+    self.power = power
+}
+
+/// Updates the sloth's energy level.
+///
+/// - Parameter energyLevel: The sloth's new energy level.
+mutating public func update(_ energyLevel: Int) {
+    self.energyLevel = energyLevel
+}
+```
+
+Both methods have an identical symbol path of `SlothCreator/Sloth/update(_:)`. 
+In this scenario, and to ensure uniqueness, DocC uses the symbol's unique 
+identifier instead of its name to disambiguate. DocC's warnings about ambiguous
+symbol links suggests one disambiguation for each of the symbols that match the
+ambiguous symbol path.
+
+```markdown
+### Updating Sloths
+- ``Sloth/update(_:)-4ko57``
+- ``Sloth/update(_:)-jixx``
+```
+
+In the example above, both symbols are functions so you need the unique 
```

Comment:
```suggestion
In the example above, both symbols are functions, so you need the unique 
```

Reply:
Resolved in https://github.com/apple/swift-docc/pull/742/commits/2e64258bcfa7eff3e8272bf1c2075245b5f4d9c5